src/script/view_model/ActionsViewModel.ts

```diff
@@ -303,22 +303,18 @@ export class ActionsViewModel {
   };
 
   readonly deleteConversation = (conversationEntity: Conversation): Promise<void> => {
-    if (conversationEntity && conversationEntity.isCreatedBySelf()) {
-      return new Promise(() => {
-        PrimaryModal.show(PrimaryModal.type.CONFIRM, {
-          primaryAction: {
-            action: () => this.conversationRepository.deleteConversation(conversationEntity),
-            text: t('modalConversationDeleteGroupAction'),
-          },
-          text: {
-            message: t('modalConversationDeleteGroupMessage'),
-            title: t('modalConversationDeleteGroupHeadline', conversationEntity.display_name()),
-          },
-        });
+    return new Promise(() => {
```

Comment:
Not sure why we wrapped it in the promise.
We should remove it and update the update the consumers.